docs/_pages/installation/roborock.md

```diff
@@ -178,7 +178,7 @@ Next, download the latest [valetudo-armv7-lowmem.upx binary](https://github.com/
 
 
 With the dustbuilder now building your image, you can start the disassembly of the robot until you reach the mainboard.
-If you need help on how to do that, there fortunately are a few videos on YouTube.
+If you need help on how to do that, there fortunately are a few videos on YouTube (e. g. [this one for Roborock S7 (German only)](https://www.youtube.com/watch?v=BGScpICKEbE)).
```

Comment:
Not having any link here is actually on purpose because links can and will turn dead, requiring the docs to be updated.
For example, a video could be deleted, replaced with something else or maybe one day we'll all shift to a different video 

Generally speaking, referencing third-party resources is best to be avoided.